Pragya

Pragsh opened the pull request:
I am not able to use commit feature because i have no authentication rights to push back to origin.help me out please.

bhankas committed:
zz

bhankas committed:
Revert "Add basics to home.html"

This reverts commit 42d2e8fa1f2a714844771d51d80906bf801ed279.

bhankas committed:
Add Registration page and link it in Home page
Initial commit for registration page.

bhankas committed:
Revert "Add Registration page and link it in Home page"

This reverts commit 5bd7c1aacfc9d3249c8a945080af1d5957503914.

bhankas committed:
Add Registration page
Added empty registration page and linked it in Home page.

bhankas committed:
Fixed typo in filename

bhankas committed:
remove old file

bhankas committed:
Prepare homepage for additions

bhankas committed:
Add jquery for easy javascript-fu

bhankas committed:
Start buiding a game

bhankas committed:
Start building a calculator

bhankas commented:
@Pragsh you have to first push commits to your own fork. Then create pull request to merge your changes here. Try watching a couple of tutorials on Youtube.

Pragsh commented:
okay :)

On Aug 6, 2017 1:56 PM, "payasrelekar" <notifications@github.com> wrote:

@Pragsh <https://github.com/pragsh> you have to first push commits to your
own fork. Then create pull request to merge your changes here. Try watching
a couple of tutorials on Youtube.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/vitragi/webdevelopment/pull/3#issuecomment-320493230>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AdPpZAYzkhzOoj8HOYCbYS86QjUl616Cks5sVXi5gaJpZM4Ore2H>
.

bhankas committed:
Add basic calculator operations

Added Addition, Subtraction, Multiplication and Division. Very skeletal layout.

bhankas committed:
Improve logic and add comments to calculator

Algorithm changed to switch, making it more efficient and more readable.
Also added comments to calculator.html
